zio-http/src/main/scala/zio/http/URL.scala

```diff
@@ -184,4 +178,79 @@ object URL {
     } yield Fragment(raw, decoded)
   }
 
+  final case class QueryParams private[http] (map: Map[String, List[String]])
```

Comment:
I think `QueryParams` should be available at `zio.http` like `Path`.

Reply:
Done